dotcom-rendering/src/layouts/FrontLayout.tsx

```diff
@@ -110,9 +114,48 @@ export const decideAdSlot = (
 			</Hide>
 		);
 	}
+
 	return null;
 };
 
+/**
+ * Renders a fronts-banner ad when in the fronts banner AB test.
+ * Only applies to network fronts on desktop screens and wider.
+ */
+export const decideFrontsBannerAdSlot = (
+	renderAds: boolean,
+	hasPageSkin: boolean,
+	isInFrontsBannerTest: boolean,
+	pageId: string,
+	collectionName: string,
+) => {
+	// We're not using contentType === "Network Front", just
+	// in case Europe goes live before the testing concludes.
+	// We don't want Europe in this test.
+	const networkFrontPageIds = ['uk', 'us', 'au', 'international'];
+	const targetedSections = frontsBannerAdSections[pageId];
+
+	if (
+		!renderAds ||
+		!isInFrontsBannerTest ||
+		!networkFrontPageIds.includes(pageId) ||
+		!targetedSections?.includes(collectionName)
+	) {
+		return null;
+	}
+
+	const adIndex = targetedSections.findIndex((ad) => ad === collectionName);
```

Comment:
Not sure if this is a problem at this point but won't this be the index in targetedSections? Not of the sections on the page, which may change?

Reply:
I think that for this test it might actually be a good thing, as when we're looking at revenue per slot, we can link slot name to section. Unless there's something I'm missing about why we'd want a typical inline ad slot naming?

Hopefully these sections won't be removed for the duration of the test. After the test, we'll use the typical naming system used for inline slots.

Reply:
Sorry, wasn't concerned about the linking slot names to sections, I brought this up as one of the Fronts DCR migration ad ratio issues was because the index's passed to the ad slot components wasn't the actual index of the ad on the page. Not sure how the index is used by fronts-banner slots but something to be aware of!

Reply:
Ah OK thanks. We're not measuring ad-ratio with this test as we know how many ads we will be serving. The index has no semantic meaning for fronts-banner slots at this moment, so we should be fine

Reply:
I know this is just a test, but I'd feel better if we used a loop counter to get the ad indexes. We'll need to do it in the future anyway when we don't have the targeted sections hard coded right?

Reply:
Yeah that's true. I have updated it to use a counter